gettsim/synthetic_data/synthetic.py

```diff
@@ -78,6 +80,9 @@ def create_synthetic_data(
     age_adults (list of int):
         Assumed age of adult(s)
 
+    gen_adults (list of int):
```

Comment:
It seems like below it is used as a boolean variable.

- Change docstring
- Probably also better to name the variable `female_adults`?

Reply:
It replaced it by gen_female. also changed the docstring.